contribute/backend/database.md

```diff
@@ -73,21 +75,21 @@ For transactions, use the `WithTransactionalDbSession` method instead.
 
 ## Migrations
 
-As Grafana evolves, it becomes necessary to create _schema migrations_ for one or more database tables.
+As your use of Grafana evolves, it may become necessary to create _schema migrations_ for one or more database tables.
```

Comment:
could this be active? You many need to create ...

Reply:
done